localstack-core/localstack/services/events/packages.py

```diff
@@ -15,12 +16,22 @@ def get_versions(self) -> list[str]:
         return [EVENT_RULER_VERSION]
 
     def _get_installer(self, version: str) -> PackageInstaller:
-        return MavenPackageInstaller(
+        return EventRulerPackageInstaller()
+
+
+class EventRulerPackageInstaller(MavenPackageInstaller):
+    def __init__(self):
+        super().__init__(
             f"pkg:maven/software.amazon.event.ruler/event-ruler@{EVENT_RULER_VERSION}",
             f"pkg:maven/com.fasterxml.jackson.core/jackson-annotations@{JACKSON_VERSION}",
             f"pkg:maven/com.fasterxml.jackson.core/jackson-core@{JACKSON_VERSION}",
             f"pkg:maven/com.fasterxml.jackson.core/jackson-databind@{JACKSON_VERSION}",
         )
 
+        self.java_version = "11"
```

Comment:
That one is a bit tricky because the JPype docs are pretty outdated and running our test suite is better than reading the docs: "JPype has been tested with Java versions from Java 1.8 to Java 13." https://jpype.readthedocs.io/en/latest/install.html

It's currently not tested in the main pipeline and the only hard dependency is EventBridge v2 (which is opt-in for now).

Reply:
Okay, but that would still mean that it's compatible with Java 11 (which is the current default version).
My point here is to avoid setting the explicit version when it's not strictly necessary (see other comments)...

Reply:
that's fine then 👍 

I thought it would be a compatibility indicator and not specifying a version means it's compatible with everything.

Reply:
Jpype will soon have annual update.  Please request anything you need updated in the docs.  As far as tested versions I believe we are testing to Java 21.  It is somewhat limited by what azure cloud supports at time of release.